test/test_contact.py::ContactTest::test_contact_page
--------------------------------------------------------------------
Last Page: https://automationexercise.com/contact_us
 Duration: 11.40s
  Browser: Edge 138.0.3351.95
   Driver: msedgedriver 138.0.3351.95
Timestamp: 1753444188  (Unix Timestamp)
     Date: Friday, 25 July 2025
     Time: 5:34:48 PM  (Nepal Standard Time, UTC+05:750)
--------------------------------------------------------------------
Traceback:
  File "C:\Users\onwei_on_cloud\PycharmProjects\SeleniumBase\test\test_contact.py", line 25, in test_contact_page
    self.assert_text("Success! Your details have been submitted successfully.", ".status.alert.alert-success")
  File "C:\Users\onwei_on_cloud\PycharmProjects\SeleniumBase\.venv\Lib\site-packages\seleniumbase\fixtures\base_case.py", line 10258, in assert_text
    self.wait_for_text_visible(text, selector, by=by, timeout=timeout)
  File "C:\Users\onwei_on_cloud\PycharmProjects\SeleniumBase\.venv\Lib\site-packages\seleniumbase\fixtures\base_case.py", line 10081, in wait_for_text_visible
    return page_actions.wait_for_text_visible(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\onwei_on_cloud\PycharmProjects\SeleniumBase\.venv\Lib\site-packages\seleniumbase\fixtures\page_actions.py", line 639, in wait_for_text_visible
    timeout_exception(TextNotVisibleException, message)
  File "C:\Users\onwei_on_cloud\PycharmProjects\SeleniumBase\.venv\Lib\site-packages\seleniumbase\fixtures\page_actions.py", line 267, in timeout_exception
    raise exc(msg)
Exception: 
 Expected text substring {Success! Your details have been submitted successfully.} for {.status.alert.alert-success} was not visible after 7 seconds!